README.md

```diff
@@ -231,13 +231,14 @@ request, and reviewers will not merge branches with tests that have
 lint errors, so you must fix any errors the lint tool reports. For
 details on doing that, see the [lint-tool documentation][lint-tool].
 
-But in the unusual case of error reports for things essential to a
+In the unusual case of error reports for things essential to a
 certain test or that for other exceptional reasons shouldn't prevent
 a merge of a test, update and commit the `lint.whitelist` file in the
-web-platform-tests root directory to suppress the error reports. For
-details on doing that, see the [lint-tool documentation][lint-tool].
+web-platform-tests root directory to suppress the error reports. 
+
+For more details, see the [lint-tool documentation][lint-tool].
 
-[lint-tool]: https://github.com/w3c/web-platform-tests/blob/master/docs/lint-tool.md
+[lint-tool]: /docs/_writing-tests/lint-tool.md
```

Comment:
http://web-platform-tests.org/writing-tests/lint-tool.html might be better, I think.